### DISCUSSION TITLE
after connecting my bluetooth device, not getting sound in bluetooth device. tried all solution available on internet but non of them working

### DISCUSSION THREAD

**Discussion by thehimanshoo** [blueman-project/blueman#2525] [Q&A]
blueman:
BlueZ:
Distribution:
Desktop environment:

* [ ] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

**Comment by infirit** [comment]
After Blueman connects the device audio is handled by PulseAudio and more recently Pipewire. 

Maybe you're best off asking on your distributions support channels.
